projects/packages/connection/src/class-manager.php

```diff
@@ -1062,6 +1062,8 @@ public function register( $api_endpoint = 'register' ) {
 			? $stats_options['blog_id']
 			: null;
 
+		$package_versions = apply_filters( 'jetpack_package_versions', array() );
```

Comment:
It may be good to add a docblock here, so we can keep track of things? For example something like:

```suggestion
		/* This action is documented in src/class-package-version-tracker.php */
		$package_versions = apply_filters( 'jetpack_package_versions', array() );
```